.github/workflows/update-generated-docs.yml

```diff
@@ -0,0 +1,32 @@
+name: Update generated docs
+
+on:
+  push:
+
+jobs:
+  Update-generated-docs:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Unshallow
+        run: git fetch --prune --unshallow
+      -
+        name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16 # please change also in `ci.yml`,`release.yml` and `go.mod`
+      -
+        name: Update generated docs
+        run: ./generate-docs.sh
+      -
+        name: Commit changes
```

Comment:
will this always create a commit?
can we do it only when the docs have changed ?

Reply:
It will Only commit it has what to commit 😄